.github/workflows/Format-check.yml

```diff
@@ -0,0 +1,11 @@
+name: Format suggestions
```

Comment:
I don't understand. This PR already replaces it, isn't it?

Reply:
Maybe we should just replace the contents of the FormatPR.yml file instead of creating a new filename.

Reply:
Ah, yes, I can do that. I read the name "FormatPR.yml" more as "create a format PR" rather than "do format a PR", which is why I thought this name is not appropriate anymore, but now that I think about it, the name is fine, I think. See [3775f44](https://github.com/JuliaGeometry/Meshes.jl/pull/965/commits/3775f44aca64e9d3c64484d81559893290b33177).